include/curl/curl.h

```diff
@@ -1067,6 +1067,7 @@ typedef CURLSTScode (*curl_hstswrite_callback)(CURL *easy,
 #define CURLPROTO_SMBS   (1<<27)
 #define CURLPROTO_MQTT   (1<<28)
 #define CURLPROTO_GOPHERS (1<<29)
+#define CURLPROTO_GEMINI (1<<30)
```

Comment:
No can do. This set of defines is already full. To add a new, add it privately in `lib/urldata.h` similar to how we add WS and WSS there.

I presume there is only ever `gemini` and no different one for TLS vs non-TLS?